ssapy/io.py

```diff
@@ -586,87 +589,13 @@ def load_b3obs_file(file_name):
     }
     return catalog
 
-######################################################################
```

Comment:
removing the MPI functions I added before as they do not really belong in ssapy. I believe better functions can be added if we want parallelization to be built into ssapy.